studio/src/mas-placeholders.js

```diff
@@ -0,0 +1,1479 @@
+                                    <span class="required">*</span></label
+                                >
+                                <sp-textfield
+                                    id="placeholder-key"
+                                    placeholder="Key"
+                                    .value=${this.newPlaceholder.key}
+                                    @input=${this.handleNewPlaceholderKeyChange}
+                                ></sp-textfield>
+                            </div>
+
+                            <div class="form-group">
+                                <label for="placeholder-locale"
+                                    >Choose Locale
+                                    <span class="required">*</span></label
+                                >
+                                <mas-locale-picker
+                                    id="placeholder-locale"
+                                    @locale-changed=${this
+                                        .handleNewPlaceholderLocaleChange}
+                                ></mas-locale-picker>
+                            </div>
+
+                            <div class="form-group">
+                                <label for="placeholder-value"
+                                    >Enter Value
+                                    <span class="required">*</span></label
+                                >
+                                <sp-textfield
+                                    id="placeholder-value"
+                                    placeholder="Value"
+                                    .value=${this.newPlaceholder.value}
+                                    @input=${this
+                                        .handleNewPlaceholderValueChange}
+                                ></sp-textfield>
+                            </div>
+                        </div>
+
+                        <div class="create-modal-actions">
+                            <sp-button
+                                variant="secondary"
+                                @click=${this.closeCreateModal}
+                            >
+                                Cancel
+                            </sp-button>
+                            <sp-button
+                                variant="accent"
+                                @click=${this.createPlaceholder}
+                            >
+                                Create
+                            </sp-button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        `;
+    }
+
+    handleLocaleChange(event) {
+        const newLocale = event.detail.locale;
+        Store.filters.value.locale = newLocale;
+        this.selectedLocale.value = newLocale;
+        Store.placeholders.list.loading.set(true);
+
+        if (this.repository) {
+            this.repository.searchPlaceholders();
+        }
+
+        this.requestUpdate();
```

Comment:
nit: this should not be needed, as it will refresh when one of the observed stores is updated.

Reply:
Removed